vault/resource_ssh_secret_backend_role_test.go

```diff
@@ -39,6 +39,7 @@ func TestAccSSHSecretBackendRole(t *testing.T) {
 		resource.TestCheckResourceAttr(resourceName, "allowed_users_template", "false"),
 		resource.TestCheckResourceAttr(resourceName, "allowed_users", ""),
 		resource.TestCheckResourceAttr(resourceName, "default_user", ""),
+		resource.TestCheckResourceAttr(resourceName, "default_user_template", "false"),
```

Comment:
I think we can avoid these by not setting the default as well

Reply:
will need to remove this once I don't set a default

Reply:
On second thought, I don't see any harm in keeping these here. What is the reason you want to avoid this assertion?

Reply:
Nevermind, we don't want to check this because we only want to do state checks for non-computed, non-default fields. Went ahead and removed these checks.